diff --git a/packages/cosmo-dycore/package.py b/packages/cosmo-dycore/package.py
index 796df0a..13a0c07 100644
--- a/packages/cosmo-dycore/package.py
+++ b/packages/cosmo-dycore/package.py
@@ -64,7 +64,7 @@ class CosmoDycore(CMakePackage):
     variant('cuda_arch',
             default='none',
             description='Build with cuda_arch',
-            values=('70', '60', '37'),
+            values=('80','70', '60', '37'),
             multi=False)
     variant('cuda', default=True, description='Build with cuda or target gpu')
     variant('gt1', default=False, description='Build with gridtools 1.1.3')
