diff --git a/packages/cosmo-dycore/package.py b/packages/cosmo-dycore/package.py
index 796df0a..66b11cd 100644
--- a/packages/cosmo-dycore/package.py
+++ b/packages/cosmo-dycore/package.py
@@ -170,6 +170,8 @@ class CosmoDycore(CMakePackage):
                 args.append('-DDYCORE_TARGET_ARCHITECTURE=gpu')
             else:
                 args.append('-DDYCORE_TARGET_ARCHITECTURE=CUDA')
+
+            args.append('-DCUDAToolkit_INCLUDE_DIR={}/include'.format(spec['cuda'].prefix))
         # target=cpu
         else:
             args.append('-DENABLE_CUDA=OFF')
