diff --git a/packages/int2lm/package.py b/packages/int2lm/package.py
index 0b5e38c..2f4006f 100644
--- a/packages/int2lm/package.py
+++ b/packages/int2lm/package.py
@@ -141,7 +141,7 @@ class Int2lm(MakefilePackage):
 
                 else:
                     env.set('MPIL',
-                            '-L' + self.spec['mpi'].prefix + ' -lmpich')
+                            '-L' + self.spec['mpi'].prefix + ' -lmpichf90')
 
         # Compiler & linker variables
         if self.compiler.name == 'pgi':
